### ISSUE TITLE
[Feature Request]:  This is by far the best UX/UI, please cooperate with Vlad's version!

### ISSUE THREAD

**Issue by bbecausereasonss** [anapnoe/stable-diffusion-webui-ux#75]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Automatic's base repo is behind and barely ever getting fixed.
Vlad has created 400+ extra commits and is developing daily. He's got the best repo by far.

Your interface with Vlads structure would be EPIC. PLEASE cooperate!

https://github.com/vladmandic/automatic

https://www.reddit.com/r/StableDiffusion/comments/12w0ox0/i_hope_that_by_a1111_is_everything_fine_best/

### Proposed workflow

See above

**Comment by BinaryQuantumSoul** [comment]
You absolutely should!
The merging between both of your forks will undoubtedly replace auto1111

**Comment by anapnoe** [comment]
Yes it is already happening we are working day and night on this

**Comment by space-nuko** [comment]
Thanks for picking up the torch, I hope both of you can Collab to make the best WebUI there is

**Comment by scpedicini** [comment]
+1 for a joint Vlad / Anapnoe version!

**Comment by anapnoe** [comment]
> Thanks for picking up the torch, I hope both of you can Collab to make the best WebUI there is

@space-nuko we are looking for people to join our effort you are most welcome if you want to join in you can contact me Eoan#9885 or @vladmandic vlado#8083 on discord

**Comment by space-nuko** [comment]
@anapnoe Hey I will definitely take you up on that offer but I can only get back to my computer in a week, will try to see you then

**Comment by space-nuko** [comment]
Sent a req btw

**Comment by anapnoe** [comment]
cool

**Comment by GeneralAwareness** [comment]
Oh, my I am elated to read this thread.

**Comment by midcoastal** [comment]
HOLY COW THIS IS EXCITING!  YAS!

**Comment by cibernicola** [comment]
is this real?
when, how, who, can I help somehow?

**Comment by vladmandic** [comment]
Hey, always looking for contributors! 😁

**Comment by cibernicola** [comment]
Tell me a clue from where to start :)

**Comment by vladmandic** [comment]
What's your discord id? I'll ping you tomorrow...

**Comment by cibernicola** [comment]
cibernicola#4627

**Comment by Averytingwong** [comment]
Is this really happening?! :D

**Comment by blueball81** [comment]
I have been waiting for their (vladmandic & anapnoe's) forks to be merged.  However, it looks the difference between the two forks is growing.  Will there any initiative branch to be created here or somewhere soon?  @anapnoe

**Comment by BinaryQuantumSoul** [comment]
> I have been waiting for their (vladmandic & anapnoe's) forks to be merged.  However, it looks the difference between the two forks is growing.  Will there any initiative branch to be created here or somewhere soon?  @anapnoe 

Actually anapnoe has a custom private branch of vladmandic and they will reveal it when he finishes but's it's a lot of work since they're adding even more functionalities. In the mean time I recommend using vlad's fork (SD.Next) since there are day to day improvements and hopefully someday the changelog will announc the merge.

**Comment by Aptronymist** [comment]
> Any news on this subject?

Afraid not, it has been dead for quite a few months, unfortunately, with nothing to show for it.
